GA/objectives.py

```
@@ -0,0 +1,361 @@
+
+    # Add the parent folder path to the system path
+    import sys
+    from pathlib import Path
+    parent_dir = Path(__file__).resolve().parent.parent
+    submodels_path = parent_dir / "Submodels"
+    sys.path.extend([str(parent_dir), str(submodels_path)])
+
+    from Submodels.output_handling import output_processing #type: ignore
+    import config #type: ignore
+
+    objectives_class = Objectives(config.DUCT_VALUES)
```

Pylint (3.3.7)

[error] No value for argument 'oper' in constructor call

(E1120)